### PR TITLE
Add link to ink-syntax-highlight in readme

### PR DESCRIPTION
**Pull request by vsashyn**
Add link to [ink-syntax-highlight](https://github.com/vsashyn/ink-syntax-highlight) component to readme.

### CODE REVIEW

**Comment by sindresorhus** [comment]
I think you need to work more on your readme. You have a typo, empty `Supported Languages` section, `cli-highlight.Theme` is not a type. You also need to write tests. In short, spend some more time on the package.

File: readme.md [hunk 1]
```diff
@@ -1748,6 +1748,7 @@ You can even inspect and change the props of components, and see the results imm
 - [ink-markdown](https://github.com/cameronhunter/ink-markdown) - Render syntax highlighted Markdown.
 - [ink-quicksearch-input](https://github.com/Eximchain/ink-quicksearch-input) - Select component with fast quicksearch-like navigation.
 - [ink-confirm-input](https://github.com/kevva/ink-confirm-input) - Yes/No confirmation input.
+- [ink-syntax-highlight](https://github.com/vsashyn/ink-syntax-highlight) - Code syntax highlight.
```
Comment (sindresorhus) [inline readme.md:1751]:
```suggestion
- [ink-syntax-highlight](https://github.com/vsashyn/ink-syntax-highlight) - Code syntax highlighting.
```

**Comment by vsashyn** [comment]
@sindresorhus Thank you for your feedback! I'll dig more into it and will update.

**Comment by vsashyn** [comment]
Readme for `ink-syntax-highlight` is improved, added snapshot test.

**Comment by sindresorhus** [comment]
Any reason you went with a prop for the code? Wouldn't this be more idiomatic:

```jsx
render(
	<SyntaxHighlight>{'
		const hello = "world"
	'}</SyntaxHighlight>
);
```

**Comment by sindresorhus** [comment]
I think the readme could use a screenshot of how the syntax highlighting looks in the terminal.

**Comment by vsashyn** [comment]
We've discussed with @vadimdemedes the option to  use `children` as a source code and agreed to use separate prop to have more explicit way to specify code and get stricter types support. Potentially, using children might require validation checks in the runtime.

Screenshot added ✅

**Comment by sindresorhus** [comment]
The text in the screenshot is way too small. You need to crank up the font size. Example: https://github.com/sindresorhus/pure The code indentation is also inconsistent.

**Comment by vsashyn** [comment]
Fixed. Such a good lesson to me regarding visual stuff... 🙂 
And thank you for your patience 🙏

**Comment by sindresorhus** [comment]
Looks better now. 👍🏻

In the future, I would recommend installing iTerm and using its minimal style for screenshots, since that doesn't include the titlebar, which gives the screenshot a cleaner look.

**Pull request merged**